examples/05-advanced-nodegroups.yaml

```diff
@@ -42,6 +42,11 @@ nodeGroups:
       nodegroup-type: very-special-science-workloads
     classicLoadBalancerNames:
       - ng3-classic-load-balancer
+    asgMetricsCollection:
```

Comment:
do you want to put all application values here as reference ?

- GroupMinSize
- GroupMaxSize
- GroupDesiredCapacity
- GroupInServiceInstances
- GroupPendingInstances
- GroupStandbyInstances
- GroupTerminatingInstances
- GroupTotalInstances